packages/edge-login-ui-react/src/modules/Login/LoginWithPin/LoginWithPin.js

```diff
@@ -17,11 +17,6 @@ import Mobile from './LoginWithPin.mobile.js'
 import Desktop from './LoginWithPin.web.js'
 
 class LoginWithPin extends Component {
-  componentWillReceiveProps (nextProps) {
```

Comment:
Where is cancelRequest called then? It needs to eventually be called so it doesn't run forever.

Reply:
It is now when the WHOLE login component is unmount. Meaning the whole login screen